### DISCUSSION TITLE
Shulker Backpack require specific Colored name

### DISCUSSION THREAD

**Discussion by TomLewis** [Zrips/CMI#5005] [Enhancement]
The reason we cannot enable global shulker boxes to be allowed as backpacks is no limit a player can fill their inventory full of shulkers and there will be no negative effects.

I suggest having an option for Shulker box backpack to require a special colored name. This way then the Backpacks need to be bought/traded/won no the server and only those specific named ones will act as backpacks.

**Comment by TomLewis** [comment]
Bump

**Comment by 456dev** [comment]
Personally I would prefer if this used some other nbt to avoid anvil conflicts/players making them non-special.
But it would need to track and ensure that the data stays even after being placed.

An alternative could be if they are backpackable, then they cannot be placed
